packs/vsphere/actions/get_vm.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "get_vm",
+    "runner_type": "run-python",
+    "description": "Retrieves the virtual machines on a vCenter Server system. It computes the union of Virtual Machine sets based on each parameter.",
```

Comment:
I mean that every container or related object leads us to the set of VMs.
At some point I am going to end up with many sets of VMs. Thus, I am going to make UNION of those sets as the result of the action.
Looking at the examples of PowerCli, I see that this command is called with one param only. Probably, this is the primary use-case.